example/spec/parallel/ingest/ingestFromPdrSpec.js

```diff
@@ -252,26 +278,29 @@ describe('Ingesting from PDR', () => {
             collectionId
           );
           expectedParsePdrOutput.granules[0].dataType += testSuffix;
-          expectedParsePdrOutput.granules[0].granuleId =
-            expectedParsePdrOutput.granules[0].granuleId.replace(
-              granuleDateString,
-              granuleIdReplacement
+
+          // update granule 2 with its granuleId
+          expectedParsePdrOutput.granules[1].dataType += testSuffix;
+          expectedParsePdrOutput.granules[1].granuleId =
+            expectedParsePdrOutput.granules[1].granuleId.replace(
+              testDataGranuleId,
+              testDataGranuleId2
             );
-          expectedParsePdrOutput.granules[0].files = expectedParsePdrOutput.granules[0].files.map((file) => {
-            file.name = file.name.replace(granuleDateString, granuleIdReplacement);
+          expectedParsePdrOutput.granules[1].files = expectedParsePdrOutput.granules[1].files.map((file) => {
+            file.name = file.name.replace(testDataGranuleId, testDataGranuleId2);
             return file;
           });
+
           expectedParsePdrOutput.pdr.name = pdrFilename;
         } catch (error) {
           beforeAllFailed = error;
         }
       });
 
       afterAll(async () => {
-        // wait for child executions to complete
         await Promise.all(
           queueGranulesOutput.payload.running
-            .map((arn) => waitForCompletedExecution(arn))
+            .map((arn) => deleteExecution({ prefix: config.stackName, executionArn: arn }))
```

Comment:
waitForCompletedExecution  [here](https://github.com/nasa/cumulus/pull/3922/files#diff-b7a6c9806248e79560530502ae9fe7443e970567d465dc7c694a5c42f5018df3R464)
and move deleteExecution both ingest workflow executions here.